Area51/Sources/ViewModels/ContributorModel.swift

```diff
@@ -5,6 +5,20 @@ struct Contributor: Decodable {
     var login: String?
     var name: String?
     var contributions: Int
+    var avatarURL: String?
+
+    enum CodingKeys: String, CodingKey {
+        case login, name, contributions
+        case avatarURL = "avatar_url"
+    }
+
+    init(from decoder: Decoder) throws {
```

Comment:
can delete this entire init block, the coding should just work with the CodingKeys specified above.